### PR TITLE
Porting Time 80 TKL

### PR DESCRIPTION
**Pull request by lukelex**
Porting Time 80 TKL PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/foxlab/key65/hotswap/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Foxlab Key 65 Hotswap
 
-![Key 65](https://i.imgur.com/28Qck8ul.jpg)
+![Key 65](https://i.imgur.com/8HpYStP.jpg)
```
Comment (zvecr) [inline keyboards/foxlab/key65/hotswap/readme.md:3]:
```suggestion
![Key 65](https://i.imgur.com/28Qck8ul.jpg)
```

File: keyboards/foxlab/time80/rules.mk [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+WS2812_DRIVER = i2c
```
Comment (zvecr) [inline keyboards/foxlab/time80/rules.mk:14]:
```suggestion
# MCU name
MCU = atmega32a

# Bootloader selection
BOOTLOADER = bootloadHID

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
WS2812_DRIVER = i2c
```

File: keyboards/foxlab/time80/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Make example for this keyboard (after setting up your build environment):
+
+    make foxlab/time80:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
+
+    make foxlab/time80:default:flash
+
+**Reset Key**: There are no reset switches or pads. Follow this
+[guide](https://beta.docs.qmk.fm/using-qmk/guides/flashing/flashing_bootloadhid)
```
Comment (zvecr) [inline keyboards/foxlab/time80/readme.md:23]:
```suggestion
[guide](https://beta.docs.qmk.fm/using-qmk/guides/flashing/flashing_bootloadhid)
```

**Review by zvecr — CHANGES_REQUESTED** [review]
You are also missing an [info.json](https://docs.qmk.fm/#/reference_info_json) for this keyboard

File: keyboards/foxlab/time80/config.h [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0000
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Lukas Alexander
+#define PRODUCT      time80
```
Comment (fauxpark) [inline keyboards/foxlab/time80/config.h:27]:
```suggestion
#define PRODUCT_ID   0x6C50
#define DEVICE_VER   0x0001
#define MANUFACTURER Fox Lab
#define PRODUCT      Time80
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/key65/hotswap/readme.md (hunk 1 above)
Comment (lukelex) [reply keyboards/foxlab/key65/hotswap/readme.md:3]: My bad. Didn't mean to change this. Thanks

File: keyboards/foxlab/time80/readme.md (hunk 3 above)
Comment (lukelex) [reply keyboards/foxlab/time80/readme.md:23]: I can't spot the difference. What changed?

File: keyboards/foxlab/time80/config.h (hunk 4 above)
Comment (lukelex) [reply keyboards/foxlab/time80/config.h:27]: > echo -n "Time80" | shasum | cut -c1-4

This is new to me. Is it documented anywhere?

File: keyboards/foxlab/time80/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# time80
+
+![time80](https://i.imgur.com/8HpYStP.jpg)
```
Comment (fauxpark) [inline keyboards/foxlab/time80/readme.md:3]:
```suggestion
![time80](https://i.imgur.com/8HpYStPl.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/time80/readme.md (hunk 3 above)
Comment (zvecr) [reply keyboards/foxlab/time80/readme.md:23]: Sorry, bad copy paste. The suggested change has been updated.

File: keyboards/foxlab/time80/readme.md (hunk 5 above)
Comment (lukelex) [reply keyboards/foxlab/time80/readme.md:3]: Smaller is better?

Comment (fauxpark) [reply keyboards/foxlab/time80/readme.md:3]: Smaller, but also smaller filesize to download.

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by lukelex** [comment]
@zvecr I understand the `info.json` is used for QMK Configurator but is it strictly necessary for merging the PR?

**Review by noroadsleft — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
> I understand the `info.json` is used for QMK Configurator but is it strictly necessary for merging the PR?

@zvecr likes them on the original PR, but I can make one if you can't be bothered.

**Comment by lukelex** [comment]
> @zvecr likes them on the original PR, but I can make one if you can't be bothered.

I'll pay you a beer.

**Comment by noroadsleft** [comment]
> I'll pay you a beer.

I actually don't drink alcohol, but I appreciate the offer.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**